guides/fcm-flutter-novu/how-to-send-push-notifications-to-flutter-apps-with-fcm-using-Novu.mdx

```diff
@@ -0,0 +1,735 @@
+
+<Frame caption="Add Push title and content"><img src="https://github.com/novuhq/docs/assets/2946769/e6f5aefb-4abe-4190-8d94-16f31650427d"/> </Frame>
+<Frame caption="Trigger Workflow"><img src="https://github.com/novuhq/docs/assets/2946769/cfa2e904-f07a-4796-9020-6917fdb17aa8"/> </Frame>
+
+**Receive Triggered Push Workflow on Device**
+
+<Frame caption="Push Notification Received on Device">
+  <img src="https://github.com/novuhq/docs/assets/2946769/741dc520-1b0e-429e-bcf8-165536ef5dd4
+"/> 
+</Frame>
+
+
+## Dynamic Content
+
+When we were creating the first workflow, we "Hardcoded" the content of the notification.
+
+Now, we will determine the content when calling the API.
+
+1. In the workflow, we should change the values of the `title` and the `body` to `{{title}}` and `{{body}}`.
+That way, we could insert values through the payload of the API call.
+
+<Frame><img src="https://github.com/novuhq/docs/assets/2946769/03c268b1-8c7a-4c8b-9a67-b4a401139bd4"/> </Frame>
+
+2. Add a `payload` object to the API call.
+
+Insert the `trigger identifier` as the name in the API request below:
+
+```JSON
+
+curl --location --request POST 'https://api.novu.co/v1/events/trigger' \
+     --header 'Authorization: ApiKey <REPLACE_WITH_API_KEY>' \
+     --header 'Content-Type: application/json' \
+     --data-raw '{
+         "name": "novu-push-workflow-for-flutter",
+         "to": {
+           "subscriberId": "584349343"
+         },
+         "payload": {
+           "title": "This title was set via the Payload",
+           "body": "This body was set via the Payload"
+         },
+       }'
+
+```
+
+<Frame caption="Push Notification received on device from Novu via FCM"><img src="https://github.com/novuhq/docs/assets/2946769/b7a2d4b2-003b-423a-856c-207072f0bed6"/> </Frame>
+
+## Handling Data Type Messages
+
+With FCM, you can send `Notification` and `Data` messages. We have handled sending notification messages. Let's handle data messages.
+
+Data messages contain user-defined custom key-value pairs. The information is encapsulated in the `data` key. The app can do whatever it pleases with the data once it's received by the device.
+
+**Via the Trigger API call**
+
+```JSON
+
+curl --location --request POST 'https://api.novu.co/v1/events/trigger' \
+     --header 'Authorization: ApiKey <REPLACE_WITH_API_KEY>' \
+     --header 'Content-Type: application/json' \
+     --data-raw '{
+         "name": "novu-push-workflow-for-flutter",
+         "to": {
+           "subscriberId": "584349343"
+         },
+         "payload": {
+           "title": "This title was set via the Payload",
+           "body": "This body was set via the Payload"
+         },
+         "overrides": {
+          "fcm": {
+            "data": {
+              "subscription_status": "active",
+              "renewed_by": "bisola"
+            }
+          }
+         }
+       }'
+
+```
+
+<Frame caption="Push & Data Notification received on device in foreground from Novu via FCM">
```

Comment:
Notification received on device in foreground => Notification received on device in the foreground